src/applications/vaos/tests/useFormPageTester.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { getDefaultFormState } from '@department-of-veterans-affairs/react-jsonschema-form/lib/utils';
+import {
+  updateSchemaAndData,
+  updateItemsSchema,
+} from 'platform/forms-system/src/js/state/helpers';
+
+export default function useFormPageTester(initialData) {
```

Comment:
This is similar to `DefinitionTester`, except we want to test page components, not plain schemas.